javascript-methods/javascript-methods-quiz-notes.md

```diff
@@ -0,0 +1,74 @@
+# javascript-methods-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do we log things to the console?
+The JavaScript console is a debugging tool. It is where the browser prints errors and warnings as they occur in your JavaScript code.
+
+- What is a method?
+a method is a function which a property of an object
+
+- How is a method different from any other function?
+A method, like a function, is a set of instructions that perform a task. The difference is that a method is associated with an object, while a function is not.
+
+- How do you remove the last element from an array?
+pop() method. The pop() method removes the last element from an array and returns that element. This method changes the length of the array.
+
+- How do you round a number down to the nearest integer?
+math.floor
```

Comment:
Because `floor()` is a method and not a property this should be `Math.floor()`. `Math` should also be capitalized as it is a special object. 

The same applies on line 23.